feathr-example/static_features.ipynb

```diff
@@ -0,0 +1,1632 @@
+      "Requirement already satisfied: requests-oauthlib>=0.5.0 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from msrest>=0.6.21->azure-storage-file-datalake>=12.5.0->feathr) (1.3.1)\n",
+      "Requirement already satisfied: isodate>=0.6.0 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from msrest>=0.6.21->azure-storage-file-datalake>=12.5.0->feathr) (0.6.1)\n",
+      "Requirement already satisfied: et-xmlfile in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from openpyxl>=3.0->pyapacheatlas->feathr) (1.1.0)\n",
+      "Requirement already satisfied: soupsieve>1.2 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from beautifulsoup4->google>=3.0.0->feathr) (2.3.2.post1)\n",
+      "Requirement already satisfied: pycparser in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from cffi>=1.12->cryptography>=2.5->azure-identity->feathr) (2.21)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0.0dev,>=1.16.0->google-api-python-client>=2.41.0->feathr) (0.4.8)\n",
+      "\u001b[33mWARNING: There was an error checking the latest version of pip.\u001b[0m\u001b[33m\n",
+      "\u001b[0mNote: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -U feathr pandavro scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite: Configure the required environment\n",
+    "\n",
+    "In the first step (Provision cloud resources), you should have provisioned all the required cloud resources. If you use Feathr CLI to create a workspace, you should have a folder with a file called `feathr_config.yaml` in it with all the required configurations. Otherwise, update the configuration below.\n",
+    "\n",
+    "The code below will write this configuration string to a temporary location and load it to Feathr. Please still refer to [feathr_config.yaml](https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml) and use that as the source of truth. It should also have more explanations on the meaning of each variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tempfile\n",
+    "yaml_config = \"\"\"\n",
+    "# Please refer to https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml for explanations on the meaning of each field.\n",
```

Comment:
Same comment about pointing to a release tag.

Reply:
Pointed this to release `v0.7.2`.